v3/operations.go

```diff
@@ -10692,6 +10692,8 @@ type CreateInstanceRequest struct {
 	SSHKeys []SSHKey `json:"ssh-keys,omitempty"`
 	// Instance template
 	Template *Template `json:"template" validate:"required"`
+	// [Beta] Enable Trusted Platform Module (TPM)
```

Comment:
Just a note here as a reminder @ste-m.

When I see this comment, I definitely think a beta field in the OA spec would be better, and egoscale would generate a beta branch version of the spec.